ERCS/erc-7579.md

```diff
@@ -223,6 +229,9 @@ If the smart account has a fallback handler installed, it:
 - MUST route to fallback handlers based on the function selector of the calldata
 - MAY implement authorization control, which SHOULD be done via hooks 
 
+If the account is adding features via fallback, in context of requirements defined by this specification, it should be considered the same as if the account was implementing those features natively.
```

Comment:
```suggestion
If the account adds features via fallback, these should be considered the same as if the account was implementing those features natively.
```

Reply:
can this lead to accounts not having these features if the right fallback isnt isntalled? is this a problem?

Reply:
I think yes, this can lead for the account to end up not having these features. It will make the account not compatible with this standard. I do not think this is a problem, however, maybe, it can be explicitly outlined above.

Something like 

> In this case, uninstalling such a fallback will lead for account to become incompatible with ERC-7579.

wdyt?

Reply:
I think yes, this can lead for the account to end up not having these features. It will make the account not compatible with this standard. I do not think this is a problem, however, maybe, it can be explicitly outlined above.

Something like 

> In this case, uninstalling such a fallback will lead for account to become incompatible with ERC-7579.

wdyt?